crowbar_framework/app/assets/javascripts/barclamps/neutron/application.js

```diff
@@ -299,6 +306,7 @@ function ml2_mechanism_drivers_check() {
     $('#ml2_type_drivers_container').hide();
     $('#ml2_type_drivers_default_provider_network_container').hide();
     $('#ml2_type_drivers_default_tenant_network_container').hide();
+    $('#vxlan_group_container').show();
```

Comment:
This is only for the multicast group setting (which only works for linuxbridge).

In the end, it doesn't matter much for now as, well, we never see the vxlan settings for linuxbridge. But this is to make this ready for when we add linuxbridge support.